### DISCUSSION TITLE
Feature Request: Pin to Always On Top

### DISCUSSION THREAD

**Discussion by James-Firth** [File-New-Project/EarTrumpet#942] [Ideas]
I've been using ear trumpet during the pandemic to chat with friends over discord while playing games.

My headset has two output devices (game and chat) so I often want to tweak volumes for different outputs.

By adding an "always on top" setting toggle it would float on top of all other windows and make it easier to change volumes quickly.

**Comment by riverar** [comment]
@James-Firth Good idea. If it helps, our dev builds have EarTrumpet Actions. That will let you assign more granular shortcuts as needed.

**Reply by James-Firth** [reply]
That very well may handle it as well! I'll have to check it out once it's in stable :)

I think this request is still worth leaving up as some people prefer visuals/mouse interactions
